docs/Installing.asciidoc

```diff
@@ -99,13 +135,66 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database
+
+Since verion _4.5.1512500474.437cc1c7_ of openQA, PostgreSQL is used as the
+default database.
+
+It is possible to configure other databases.
```

Comment:
Well, it's not … assuming that "other databases" mean some other than postgreSQL

Reply:
That's it what I was told... (@foursixnine ?)
Then removing the line.